src/Components/Backoffice/Home/Edit.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import { useFormik } from "formik";
+import Button from "@material-ui/core/Button/Button";
+import EditItem from "./EditItem";
+
+const Edit = (props) => {
+  const { homeWelcomeTitle } = props;
+  const [slides, setSlides] = useState([]);
+  const [slidesIds, setSlidesIds] = useState([]);
+
+  useEffect(() => {
+    async function getUser() {
```

Comment:
this should be on the SlidesService

Reply:
also use .catch(()=>{}) instead of try/catch

Reply:
Done